M2/Macaulay2/packages/Macaulay2Doc/functions/setupPromote-doc.m2

```diff
@@ -0,0 +1,29 @@
+--- status: DRAFT
+--- author(s): PZJ
+--- notes: 
+
+undocumented methods setupPromote
+
+doc ///
+ Key
+  setupPromote
+ Headline
+  automatic promotion from one ring to another
```

Comment:
Your force push made my comment disappear, but still, I don't think this headline describes what this function does. If you don't want to remove "automatic", at least add "set up" to the beginning? Also:
- list at least one method key
- maybe link to this node from `promote`? (and similarly from `lift` to `setupLift`)

Reply:
will rewrite doc